README.md

```diff
@@ -23,14 +23,149 @@ Or install it yourself as:
 
     $ gem install bitreserve
 
-## Usage
+# Contents
 
-TODO: Write usage instructions here
+* [1. Usage](#1-usage)
+* [2. Authentication](2#-authentication)
+  * [2.1 OAuth2](#21-oauth2)
+  * [2.2 Basic Auth](#21-basic-auth)
+  * [2.4 Personal Access Token](#24-personal-access-token)
+    * [2.4.1 Via argument](#241-via-argument)
+    * [2.4.2 Via environment variable](#242-via-environment-variable)
+* [3. Entities](#3-entities)
+* [4. API calls](#4-endpoints)
+  * [4.1 Currencies](#41-currencies)
+  * [4.2 Tickers](#42-tickers)
+  * [4.4 Entities](#43-entities)
+  * [4.4 Cards](#44-cards)
+  * [4.5 Transactions](#45-transactions)
+  * [4.6 Contacts](#46-contacts)
+  * [4.7 Users](#47-users)
+  * [4.8 Transparency](#48-transparency)
+* [5. Testing](X#-testing)
+* [6. Contributing](X#-contributing)
 
-## Contributing
+# 1 Usage
```

Comment:
I'm not very fond of have the numbers here. I'd prefer this to be just "Usage". In general I prefer the READMEs to be easy going in language and flow, so that they are also easier to read. In that regard I wouldn't usually have a table of contents, but I don't oppose it. However, this numbering and some other minor things make it a bit too serious and enterprisey for my taste, at least.